server/game/cards/08.1-TAK/BloodyArakh.js

```diff
@@ -0,0 +1,36 @@
+const DrawCard = require('../../drawcard.js');
+
+class BloodyArakh extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            max: ability.limit.perPhase(1),
+            when: {
+                afterChallenge: event => event.challenge.winner === this.controller && event.challenge.challengeType === 'military' &&
+                                         event.challenge.isParticipating(this.parent)
+            },
+            handler: context => {
+                context.player.sacrificeCard(this);
+
+                this.untilEndOfPhase(ability => ({
+                    targetType: 'player',
+                    effect: ability.effects.modifyChallengeTypeLimit('military', 1)
+                }));
+
+                this.game.addMessage('{0} sacrifices {1} and is then able to initiate an additional {2} challenge this phase',
```

Comment:
proposed simpler wording "… and can initiate an additional …"

Reply:
That's probably better. I've always tried to signify 'then'-effects by including the word, but they keep introducing weirder effects making for such awakward messages.